automated/linux/docker-prune/disable_prune.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
```

Comment:
maybe change this file to disable-prune.sh same with the yaml file?
There are more files with "-" instead of "_"....